analyzers/tests/SonarAnalyzer.UnitTest/TestCases/MagicNumberShouldNotBeUsed.cs

```diff
@@ -36,10 +86,23 @@ public override int GetHashCode()
             return MY_VALUE * 397;
```

Comment:
should also add `TimeSpan.FromMinutes` and generally ignore if it's with `TimeSpan.From...` because it's a common usecase